docs/src/references/c8y-configuration-management.md

```diff
@@ -264,4 +271,75 @@ When there is no coresponding config file on the thin-edge device the plugin sen
   * external device `child1`: recognizes the MQTT notification message downloads the new configuration file with HTTP GET and the URL from the thin-edge device
   * external device `child1`: processes the new configuration
 
+--------------------------------------------------------------------------------
+
+**Proposal 3:**
```

Comment:
Having proper section titles would be better. Both markdown and wording can be improved.

```suggestion
### Deliver and update configuration files using a thin-edge HTTP service
```

Reply:
Improved now, see https://github.com/cstoidner/thin-edge.io/commit/0e2f15737d6652ba84aebf475bf710d3d560099f